pkgs/applications/networking/instant-messengers/dino/default.nix

```diff
@@ -89,35 +84,20 @@ stdenv.mkDerivation (finalAttrs: {
     gst-plugins-good # contains rtpbin, required for VP9
     gst-plugins-bad # required for H264, MSDK
     gst-vaapi # required for VAAPI
-    webrtc-audio-processing
+    webrtc-audio-processing_1
   ];
 
-  cmakeFlags = [
-    "-DBUILD_TESTS=true"
-    "-DRTP_ENABLE_H264=true"
-    "-DRTP_ENABLE_MSDK=true"
-    "-DRTP_ENABLE_VAAPI=true"
-    "-DRTP_ENABLE_VP9=true"
-    "-DVERSION_FOUND=true"
-    "-DVERSION_IS_RELEASE=true"
-    "-DVERSION_FULL=${finalAttrs.version}"
-    "-DXGETTEXT_EXECUTABLE=${lib.getBin buildPackages.gettext}/bin/xgettext"
-    "-DMSGFMT_EXECUTABLE=${lib.getBin buildPackages.gettext}/bin/msgfmt"
-    "-DGLIB_COMPILE_RESOURCES_EXECUTABLE=${lib.getDev buildPackages.glib}/bin/glib-compile-resources"
-    "-DSOUP_VERSION=${lib.versions.major libsoup_2_4.version}"
+  mesonFlags = [
+    "-Dplugin-notification-sound=enabled"
+    "-Dplugin-rtp-h264=enabled"
+    "-Dplugin-rtp-msdk=enabled"
+    "-Dplugin-rtp-vaapi=enabled"
+    "-Dplugin-rtp-vp9=enabled"
   ];
 
   # Undefined symbols for architecture arm64: "_gpg_strerror"
   NIX_LDFLAGS = lib.optionalString stdenv.hostPlatform.isDarwin "-lgpg-error";
 
-  doCheck = true;
-  checkPhase = ''
-    runHook preCheck
-    ./xmpp-vala-test
-    ./signal-protocol-vala-test
-    runHook postCheck
-  '';
```

Comment:
Any particular reason why checks were dropped?

Reply:
Need to set doCheck = true and then meson check is run.